### PR TITLE
board: add cuav 7-nano

### PR DESCRIPTION
**Pull request by cuav-liu1**
Added support for CUAV-7-Nano flight controller.

### CODE REVIEW

File: boards/cuav/7-nano/pwm_voltage/pwm_voltage.cpp [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+#include <inttypes.h>
+#include <stdint.h>
+
+#include <px4_platform_common/getopt.h>
+#include <px4_platform_common/module_params.h>
+#include <uORB/topics/actuator_outputs.h>
```
Comment (dagar) [inline boards/cuav/7-nano/pwm_voltage/pwm_voltage.cpp:39]: not used
```suggestion
```

File: boards/cuav/7-nano/pwm_voltage/pwm_voltage.cpp [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	} else {
+		VDD_5V_PWM_EN(false);
+	}
+}
+
+extern "C" int pwm_voltage_main(int argc, char *argv[])
+{
+	const char *verb = argv[1];
+
+	if (!strcmp(verb, "start")) {
```
Comment (dagar) [inline boards/cuav/7-nano/pwm_voltage/pwm_voltage.cpp:61]: This isn't actually starting anything though?

Maybe the command itself should be something like `pwm_voltage_apply` and not require any arguments?

File: boards/cuav/7-nano/pwm_voltage/pwm_voltage.cpp (hunk 1 above)
Comment (cuav-liu1) [reply boards/cuav/7-nano/pwm_voltage/pwm_voltage.cpp]: I had modified it in the latest force-push. Could you check?

**Comment by cuav-liu1** [comment]
I had modified it in the latest force-push Modified mtd net configuration。

**Comment by cuav-liu1** [comment]
I had open netman in the latest force-push

**Review by dagar — APPROVED** [review]

**Comment by dagar** [comment]
There's a trivial merge conflict (.github/workflows/compile_nuttx.yml deleted) that I'm unable to resolve in place due to your repository permissions. Can you rebase or merge in latest main to resolve?

File: .github/workflows/compile_nuttx.yml [hunk 3]
```diff
@@ -34,6 +34,7 @@ jobs:
           cuav_can-gps-v1,
           cuav_nora,
           cuav_x7pro,
+          cuav_7-nano,
```
Comment (dagar) [inline .github/workflows/compile_nuttx.yml:37]:
```suggestion
```

**Comment by cuav-liu1** [comment]
Already merged in the latest main. @dagar

**Comment by cuav-liu1** [comment]
can you merge it? Thanks you!  @dagar

**Review by dagar — APPROVED** [review]

**Comment by mrpollo** [comment]
FYI, I built and flashed locally via the command line and used the MAVLink console to inspect everything, which was initialized as expected via QGC Daily.

Thanks for the merge @dagar

**Pull request merged**